docs/src/tutorials/tutorial_2.md

```diff
@@ -0,0 +1,125 @@
+```
+
+Also note that in the code above,`newvalues` must be the same size and type (or be able to convert to the type) of the old values stored in that parameter.
+
+If a user wishes to alter the connections within an existing model, `disconnect_param!` and `connect_param` can be used in conjunction with each other to update the connections within the model, although this is more likely to be done as part of larger changes involving components themslves, as discussed in the next subsection.
+
+## Parametric Modifications: DICE Example
+
+### Step 1. Download DICE
+
+The first step in this process is downloading the DICE model.  Open a Julia REPL (here done with the alias command `Julia`) and navigate to the folder where you would like to download DICE.
+```
+Julia 
+cd("directory")
+```
+Next, clone the existing FUND repository from Github and enter the new repository.
+```
+git clone https://github.com/anthofflab/mimi-dice-2010.jl.git
+cd("mimi-dice-2010.jl")
+```
+You have now successfully downloaded FUND to your local machine.
+
+### Step 2. Run DICE
+
+The next step is to run DICE.  If you wish to first get more aquainted with the model itself, take a look at the provided online documentation.  
+
+In order to run DICE, you will need to navigate to the source code folder, labeled `src`, and run the main fund file `dice2010.jl`.  This file defines a new [module](https://docs.julialang.org/en/v1/manual/modules/index.html) called `Dice2010`, which exports the function `construct_dice`, a function that returns a version of dice allowing for user specification of parameters.  Note that in order to allow access to the module, we must call `using .Dice2010`, where `.Dice2010` is a shortcut for `Main.Dice2010`, since the `Dice2010` module is nested inside the `Main` module. After creating the model `m`, simply run the model using the `run` function.
+
+```
+include("src\dice2010.jl")
+using .Dice2010
+m = construct_dice()
+run(m)
+```
+
+Note that these steps should be relatively consistent across models, where a repository for `ModelX` should contain a primary file `ModelX.jl` which exports, at minimum, a function named something like `getModelX` or `construct_ModelX` which returns a version of the model, and can allow for model customization within the call.
+
+In this case, the function `construct_dice` has the signature
+``` 
+construct_dice(params=nothing)
+```
+Thus there are no required arguments, although the user can input `params`, a dictionary definining the parameters of the model. 
+
+### Step 3. Altering Parameters
+
+In the case that you wish to alter an exogenous parameter, you may use the `update_param!` function.  For example, in DICE the parameter `fco22x` is the forcings of equilibrium CO2 doubling (Wm-2), and exists in the components `climatedynamics` and `radiativeforcing`.  If you wanted to change this value from its default value of `3.200` to `3.000` in both components,you would use the following code:
```

Comment:
Since W per square meter doesn't render nicely, you might want to spell it out rather than writing "Wm-2" which may be confusing.